frontend/src/components/Forum/mockData.tsx

```diff
@@ -0,0 +1,55 @@
+import {ForumThread, User} from "../../api";
+
+const headers = ['Tytuł tematu', 'Liczba odpowiedzi', 'Założyciel', 'Data założenia','Akcje'];
+const classes = ['title', 'num', 'username', 'date', 'action'];
+
+const content = (thread: ForumThread) => [
+    {
+        outsideClass: classes[0],
+        insideClass: 'not-last',
+        content: thread.title
+    },
+    {
+        outsideClass: classes[1],
+        insideClass: 'not-last',
+        content: thread.forumPosts.length
+    },
+    {
+        outsideClass: classes[2],
+        insideClass: 'not-last',
+        content: thread.creator.username
+    },
+    {
+        outsideClass: classes[3],
+        insideClass: 'not-last',
+        content: thread.dateCreated.getFullYear()+"-"+(thread.dateCreated.getMonth()+1)+"-"+thread.dateCreated.getDate()
+    },
+    {
+        outsideClass: classes[4],
+        insideClass: 'not-last',
+        content: null
```

Comment:
Nie wczytywałem się ale czy to not-last miało być wszędzie? Jeśli tak to czemu tego nie zahardcodować w pliku .tsx.

Ta sama uwaga z mockData, ten plik ma docelowo zniknąć, jeśli funkcja ma zostać to nie tutaj.

Reply:
Masz rację, `insideClass` to pozostałość o której zapomniałem. Dzięki za zwrócenie uwagi. Wymienione przez Ciebie rzeczy zostały przeniesione do `constants.ts`.